src/lib/models/outoforder/Core.cc

```diff
@@ -164,7 +164,8 @@ void Core::tick() {
 
   flushIfNeeded();
   fetchUnit_.requestFromPC();
-  isa_.updateSystemTimerRegisters(&registerFileSet_, ticks_);
+  isa_.updateSystemTimerRegisters(&registerFileSet_, ticks_,
+                                  writebackUnit_.getInstructionsWrittenCount());
```

Comment:
Similar to the comment in the emulation core, for the number of retired instructions, it should be `reorderBuffer_.getInstructionsCommittedCount()`

Reply:
See above